.github/workflows/api-release.yml

```
@@ -0,0 +1,38 @@
+      java-version: "21"
+      distribution: "temurin"
+      cache: "maven"
+      server-id: "central"
+      server-username: CENTRAL_USERNAME
+      server-password: CENTRAL_PASSWORD
+      gpg-private-key: ${{ secrets.gpg_private_key }}
+      gpg-passphrase: SIGN_KEY_PASS
+  - name: Perform maven release
+    run: >
+      mvn -B -ntp release:prepare release:perform -f ./ehrenamt-justiz-api/pom.xml
+      -DreleaseVersion=${{ github.event.inputs.release-version }} 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)